README.md

```diff
@@ -1,35 +1,466 @@
+    - [In-app Messaging Deeplinks](#in-app-messaging-deeplinks)
+  - [iOS Push Notifications](#ios-push-notifications)
+    - [Creating and uploading your iOS Cert](#creating-and-uploading-your-ios-cert)
+    - [Provisioning your app](#provisioning-your-app)
+  - [Android Push Notifications](#android-push-notifications)
+    - [Using a custom `GcmReceiver`](#using-a-custom-gcmreceiver)
+    - [Custom `MainActivity`](#custom-mainactivity)
+    - [No custom `MainActivity`](#no-custom-mainactivity)
+    - [Initializing the SDK](#initializing-the-sdk)
+    - [Advanced Android Push](#advanced-android-push)
+  - [Integrating Resource A/B Testing](#integrating-resource-ab-testing)
+  - [Testing your integration](#testing-your-integration)
+- [Upgrade Instructions](#upgrade-instructions)
+- [How to run the demo](#how-to-run-the-demo)
+- [How to build the SDK](#how-to-build-the-sdk)
+- [Contributing](#contributing)
+- [License](#license)
+
 Getting started
----------------
-Have a look at the quick integration guide at http://docs.swrve.com/developer-documentation/37926233/
+=
 
 Requirements
-------------
+-
+Swrve supports Unity version 4.0.0 or later on Android and iOS only. **Swrve Unity SDK is not currently supported on Unity Web Player or Windows 8.**
+
 ### Unity 4.0.0+
 The SDK needs a Unity version higher than 4.0.0 to be able to compile.
 
 ### Unity 5+ Personal or Unity 4.0.0+ Pro
 Some of the features like push notifications require the use of native plugins. Native plugins are now available in Unity Personal 5+ or previvous versions of Unity Pro.
 
+Installation Instructions
+=
+1. Download the [latest release](https://github.com/Swrve/swrve-unity-sdk/releases/latest)
+
+2. Unzip the Unity SDK, import the `Swrve.unityPackage` into your project and add the `SwrvePrefab` component to your scene.
+
+  ![Import swrve.unityPackage](/docs/images/name12.jpg)
+
+  ![Add Prefab to Scene](/docs/images/name13.jpg)
+
+3. Initialize the Swrve Component
+
+  There are two methods to initialize the SDK component. Automatic initialization and manual initialization. In almost all cases, you should you manual initialization and ensure that the "initialise on start" property is not set in the Swrve prefab.
+
+  Set the `app_id` and `api_key` fields of the component to your Swrve App ID and Swrve API Key.
+
+  ```
+  // By default, Swrve stores all customer data and content in our US data center. 
+  //SwrveComponent.Instance.Config.SelectedStack = Swrve.Stack.EU;
+
+  SwrveComponent.Instance.Init(<app_id>, "<api_key>");
+  ```
+
+  You can now compile and run your project in Unity.
+
+
+4. Go to Edit > Project Settings > Script Execution Order and move `SwrveComponent` to a higher priority in the list. This prevents you from getting a `NullReferenceException` that can occur if Swrve is called before the SDK has finished initializing.
+
+### Sending Events ###
+
+```
+SwrveComponent.Instance.SDK.NamedEvent("my_first_event");
```

Comment:
I hate 'my_first_event'  Unity is a game SDK so let's use a game example, maybe Tutorial.Start.  Then add a note about 'dot' syntax.

SwrveComponent.Instance.SDK.NamedEvent("Tutorial.Start");

Rules for sending events:

* Do not send the same named event in differing case.  We recommend camel case.
* Do not add unique identifiers to event names.  For example, Tutorial.Start.ServerID-ABDCEFG
* Do not add timestamps to event names.  For example, Tutorial.Start.1454458885
* Use '.'s in your event name to organize their layout in the Swrve dashboard.  Each '.' creates a new tree in the UI which groups your events so they are easy to locate.
* Do not send more than 1000 unique named events.
* Do not send events that start with 'swrve.'  They will be rejected